subsys/net/lib/lwm2m/Kconfig.ucifi

```diff
@@ -25,4 +25,88 @@ config LWM2M_UCIFI_BATTERY_INSTANCE_COUNT
 
 endif # LWM2M_UCIFI_BATTERY
 
+menuconfig LWM2M_UCIFI_LPWAN
+	bool "uCIFI LPWAN (3412) Support"
+	help
+	  The uCIFI LPWAN object provides attributes to monitor and maintain
+	  Low-Power Wide Area Networks.
+
+if LWM2M_UCIFI_LPWAN
+
+config LWM2M_UCIFI_LPWAN_INSTANCE_COUNT
+	int "Maximum # of uCIFI LPWAN object instances"
+	depends on LWM2M_UCIFI_LPWAN
```

Comment:
```suggestion
```

All these configuration parameters are already under `if LWM2M_UCIFI_LPWAN` so they don't need separate "depends on` clause.

Reply:
This has been fixed too.
Thanks